### ISSUE TITLE
innoextract crash: StackHash, 0xc0000005

### ISSUE THREAD

**Issue by hurda** [dscharrer/innoextract#48]
innoextract 1.5

Files to extract:
http://www.surroundbyus.com/pub/audiomuxer/AudioMuxer_0.9.6.0_Update1_Setup.exe (Inno 5.5.6)
http://www.surroundbyus.com/pub/audiomuxer/AudioMuxer_0.9.5.4_Setup.exe (Inno 5.5.0)

Listing the files in the installer works, but testing or extracting fails:

```
>innoextract.exe -e -d c:\temp\audiomuxer AudioMuxer_0.9.5.4_Setup.exe"
Extracting "AudioMuxer 0.9.5.4" - setup data version 5.5.0
 - "tmp\isxdl.dll" [temp] (121 KiB)
 - "app\AudioMuxer.exe" (<e-009 MiB)
 - "app\AudioMuxer EULA.rtf" (E.NKe-008 KiB)
 - "userappdata\AudioMuxer\Eac3to\avcodec-54.dll" (=e-009 MiB)
 - "userappdata\AudioMuxer\Eac3to\avutil-52.dll" (175 KiB)
[===============>                                           ]  0.0%    27 MiB/s
>
```

Minidumps will be sent per mail.

**Comment by dscharrer** [comment]
Thanks for the report and minidumps!

I have found an issue in my build setup resulted in the compiler adding AVX instructions to the release binaries, resulting in crashes on Windows XP and systems with older processors.

Please try this updated build and let me know if there are still any issues:
 http://constexpr.org/innoextract/files/snapshots/innoextract-1.6-rc1-windows.zip

Particularly, let me know if
- it still crashes
- the file sizes in the output are still messed up

**Comment by dscharrer** [comment]
This has been fixed in innoextract 1.6.
